### PR TITLE
Add SQLite support for IGAU Codex Discovery shipping

### PR DESCRIPTION
**Pull request by Elite-IGAU**
Update EDCommander.cs
Update SQLiteConnectionUse.cs

### CODE REVIEW

File: EliteDangerous/DB/EDCommander.cs [hunk 1]
```diff
@@ -136,19 +136,19 @@ public static void Delete(EDCommander cmdr)
 
         public static EDCommander Create(EDCommander other )
         {
-            return Create(other.name, other.EdsmName, other.EDSMAPIKey, other.JournalDir, other.syncToEdsm, other.SyncFromEdsm, 
-                                other.SyncToEddn, other.SyncToEGO, other.EGOName, other.EGOAPIKey, other.SyncToInara, other.InaraName, other.InaraAPIKey);
+            return Create(other.name, other.EdsmName, other.EDSMAPIKey, other.JournalDir, other.syncToEdsm, other.SyncFromEdsm,
+                                other.SyncToEddn, other.SyncToEGO, other.EGOName, other.EGOAPIKey, other.SyncToInara, other.InaraName, other.InaraAPIKey, other.SyncToIGAU);
```
Comment (klightspeed) [inline EliteDangerous/DB/EDCommander.cs]:
```
DB\EDCommander.cs(140,155): error CS1503: Argument 14: cannot convert from 'bool' to 'string' [C:\projects\eddiscovery\EliteDangerous\EliteDangerous.csproj]
```

Try:
```
... , toigau: other.SyncToIGAU
```

**Comment by klightspeed** [comment]
There are a lot of unnecessary whitespace changes.

**Comment by Elite-IGAU** [comment]
I think Atom / GitHub Desktop introduces the whitespace.  I had the same problem on another project I'm working on.

File: EliteDangerous/DB/EDCommander.cs (hunk 1 above)
Comment (Elite-IGAU) [reply EliteDangerous/DB/EDCommander.cs]: I'm not quite sure what you mean.  Should this be line 139/140?

            return Create(other.name, other.EdsmName, other.EDSMAPIKey, other.JournalDir, other.syncToEdsm, other.SyncFromEdsm,
                                other.SyncToEddn, other.SyncToEGO, other.EGOName, other.EGOAPIKey, other.SyncToInara, other.InaraName, other.InaraAPIKey, toigau: other.SyncToIGAU);

I was looking at how EGO and EDSM were being handled.

Comment (klightspeed) [reply EliteDangerous/DB/EDCommander.cs]: > Should this be line 139/140?
> ```
>         return Create(other.name, other.EdsmName, other.EDSMAPIKey, other.JournalDir, other.syncToEdsm, other.SyncFromEdsm,
>                             other.SyncToEddn, other.SyncToEGO, other.EGOName, other.EGOAPIKey, other.SyncToInara, other.InaraName, other.InaraAPIKey, toigau: other.SyncToIGAU);
> ```

Yes.

The parameters after `toinara` are:
```
string inaraname = null,
string inaraapikey = null,
string homesystem = null,
float mapzoom = 1.0f,
bool mapcentreonselection = true,
int mapcolour = -1,
bool toigau = false
```

Therefore `other.SyncToIGAU` was mapping to `homesystem`, where you want to map it to `toigau`

**Pull request closed without merging**